apps/web/components/AdditionalCalendarSelector.tsx

```diff
@@ -65,6 +65,10 @@ const AdditionalCalendarSelector = ({ isLoading }: AdditionalCalendarSelectorPro
                 return {
                   ...defaultStyles,
                   borderRadius: "2px",
+                  "@media only screen and (min-width: 640px)": {
```

Comment:
I miss Tailwind when things like this have to be done 😓

Reply:
can you not do that with tailwind? with `sm:max-w-xs` ?

Reply:
or does the Select not take classNames?

Reply:
exactly, it doesn't take classes from a specific part of the component, I could add a class to the whole thing and add tailwind classes with `@apply` from our custom css files, but that somehow didn't feel more maintainable than this 😅

Reply:
good point!